client/keys/utils.go

```diff
@@ -60,9 +60,16 @@ func printKeyringRecords(w io.Writer, records []*cryptokeyring.Record, output st
 }
 
 func printTextInfos(w io.Writer, kos []cryptokeyring.KeyOutput) {
-	out, err := yaml.Marshal(&kos)
-	if err != nil {
-		panic(err)
+	for _, ko := range kos {
+		if ko == (cryptokeyring.KeyOutput{}) {
```

Comment:
See [comment](https://github.com/cosmos/cosmos-sdk/pull/9695/commits/cf3e898c627c5fd054e66b955d9d132415b95ecb#r705228168). Hence,  I have to print out non empty `KeyOutput` structs only.